pkg/router/template/router.go

```diff
@@ -450,11 +458,38 @@ func (r *templateRouter) AddRoute(id string, route *routeapi.Route, host string)
 		}
 	}
 
+	var err error
+	if r.validateRouteTLSConfig {
```

Comment:
Yeah, its specific to the template router. F5 reports problems and ignores the config if a cert/key/ cacert etc is bad - does this both at admission time (config) or when it gets used (at run time).